physics/module_sf_mynn.F90

```diff
@@ -1041,6 +1170,7 @@ SUBROUTINE SFCLAY1D_mynn(flag_iter,                            &
          WSPD(I) = MAX(WSPD_ice,WSPD_wat)
          WSPD(I) = MAX(WSPD_lnd,WSPD(I))
 
+#ifndef _OPENACC
```

Comment:
Maybe this is a comment for the scheme developer @joeolson42, but there are print/write statements sprinkled in throughout this scheme, leading @timsliwinski-noaa to put in pre-processor directives to guard against printing them for GPU applications. Some of these are conditional on the state (e.g. if Temp > blahblah), others are controlled with a flag, like this one here.
To avoid needing a pre-processor directive we could control all the debug information with a debug flag, which is set by the host, and not when using GPUs. Thoughts?

Reply:
@joeolson42 Exactly.